service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -104,6 +119,31 @@ void shouldThrowAnExceptionIfCaseDoesNotContainHearings() {
                 .hasMessage("No hearings in the case");
         }
 
+
```

Comment:
vv minor, this method could be removed from nestings and used accross all tests rather than duplicating